pycode/epidemiology/epidata/getVaccinationData.py

```diff
@@ -75,13 +74,15 @@ def create_intervals_mapping(from_lower_bounds, to_lower_bounds):
             #       if no: add the remaining part 1-x to j+1
             #       if yes: compute the corresponding share and go through it iteratively
             share = 0
+            shareSum = 0
             # if not, the remaining share will be assigned all ages j
             while from_lower_bounds[i+1] > to_lower_bounds[j+1]:
-                share += (to_lower_bounds[j+1] - to_lower_bounds[j]
-                          ) / (from_lower_bounds[i+1] - from_lower_bounds[i])
+                share = (to_lower_bounds[j+1] - to_lower_bounds[j]
+                         ) / (from_lower_bounds[i+1] - from_lower_bounds[i])
+                shareSum += share
                 from_to_mapping[i].append([share, j])
                 j += 1
-            from_to_mapping[i].append([1-share, j])
+            from_to_mapping[i].append([1-shareSum, j])
```

Comment:
Is this correct? Was the 'share' the limitation of the function as it was before?

Please get in touch with me before resolving this comment !

Reply:
No i accidently left this in. I changed to as it was before but i think it is still wrong. See Issue #139

Reply:
I accidentally left this in as I thought this was a quick fix. Changed it back. Otherwise see issue #139